### ISSUE TITLE
Issue w/ Install Script

### ISSUE THREAD

**Issue by ddlidded** [LabKey/server#952]
When attempting to use the automated Linux installation script. It fails to download the distribution correctly due to a permission error stating access denied. Is there an alternative link that I can use / amend in the script to remedy this? Below is the URL that it generated.

--2024-12-12 04:06:59--  https://lk-binaries.s3.us-west-2.amazonaws.com/downloads/release/community/21.7.1/LabKey21.7.1-2-community-embedded.tar.gz
Resolving lk-binaries.s3.us-west-2.amazonaws.com (lk-binaries.s3.us-west-2.amazonaws.com)... 52.218.219.73, 52.92.232.66, 52.92.186.2, ...
Connecting to lk-binaries.s3.us-west-2.amazonaws.com (lk-binaries.s3.us-west-2.amazonaws.com)|52.218.219.73|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2024-12-12 04:06:59 ERROR 403: Forbidden.

**Comment by labkey-stuartm** [comment]
The install-script supports passing in environment or command line options to override the defaults. 
 You can use the following commands to add the necessary environment variables to your shell to install LabKey Community Edition V24.11.0. 

```
export LABKEY_DIST_URL="s3://lk-binaries/downloads/release/community/24.11.0/LabKey24.11.0-2-community.tar.gz"
export LABKEY_DIST_FILENAME="LabKey24.11.0-2-community.tar.gz"
export LABKEY_VERSION="24.11.0"
export LABKEY_DISTRIBUTION="community"
```
 Then run the installer
 `./install-labkey.bash`
 
 The script supports many other environment variables to override the defaults.  You will find more information on the variables [documented here](https://github.com/FDA-MyStudies/install-script/blob/develop/README.md#inputs-reference).

**Comment by labkey-stuartm** [comment]
I just merged a PR that had some minor fixes to reflect changes for v24.11.0 and sets the defaults for v24.11.0

If you pull the latest changes, and use one of the install examples from the README, you will get a working LabKey Server running v24.11.0.
